Domains/HairdresserDomain/Aggregate/Booking.cs

```diff
@@ -36,4 +36,9 @@ public Booking(DateTime appointmentDate, Guid userId, Guid hairdresserId)
 
     [Column(TypeName = "bit")]
     public bool IsCompleted { get; set; }
+
+    public void SetAsCompleted()
```

Comment:
Can be refactored to a single line
`public void SetAsCompleted() => IsCompleted = true`